### PR TITLE
Adapting to SDK v1.0.0

### PR DESCRIPTION
**Pull request by aaitor**
## Description

- Adaptation to SDK v1.0.0-rc0

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

File: .github/workflows/testing-new-gateway.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           repository: nevermined-io/node
+          rel: 'develop' # TODO: Remove when everything is in main branch
```
Comment (r-marques) [inline .github/workflows/testing-new-gateway.yml:35]: I think this should be `ref`

Comment (aaitor) [reply .github/workflows/testing-new-gateway.yml]: you are right

File: test/ConsumeAssetProof.test.ts [hunk 2]
```diff
@@ -1,118 +1,121 @@
...
-  });
+    const assetAttributes = AssetAttributes.getInstance({
+      metadata
+    })        
+
+    ddo = await nevermined.assets
+      .create(assetAttributes, publisher)
+
+    // console.log(`Asset registered: ${ddo.id}`)
+    // console.log(JSON.stringify(ddo.findServiceByType('metadata').attributes))
```
Comment (r-marques) [inline test/ConsumeAssetProof.test.ts:66]: cleanup

File: test/tsconfig.json [hunk 3]
```diff
@@ -1,10 +1,49 @@
...
+// {
+//   "compilerOptions": {
+//     "moduleResolution": "Node",
+//     "lib": ["es6", "es7", "dom"],
+//     "noUnusedLocals": true,
+//     "esModuleInterop": true,
+//     "target": "ES2020",
+//     "allowSyntheticDefaultImports": true
+//   }
+// }
```
Comment (r-marques) [inline test/tsconfig.json:11]: cleanup

**Review by r-marques — APPROVED** [review]

**Review by r-marques — APPROVED** [review]

**Pull request merged**